File: problems/2335-minimum-amount-of-time-to-fill-cups/solution.py
class Solution:
    def fillCups(self, amount: List[int]) -> int:
        amount.sort()
        a, b, c = amount[0], amount[1], amount[2]
        if a + b <= c:
            return c
        else:
            return (a + b + c + 1) // 2

File: problems/2335-minimum-amount-of-time-to-fill-cups/solution.cpp
#include <algorithm>
#include <vector>

class Solution {
public:
    int fillCups(std::vector<int>& amount) {
        std::sort(amount.begin(), amount.end());
        int a = amount[0];
        int b = amount[1];
        int c = amount[2];
        if (a + b <= c) {
            return c;
        }
        return (a + b + c + 1) / 2;
    }
};